src/main/java/game/pojo/Country.java

```diff
@@ -10,17 +10,17 @@ public class Country {
     int id;
     String name;
     Continent continent;
-    List<Integer> neighbourIdList;
+    List<Integer> neighborIdList;
```

Comment:
I have used the British "neighbour". Thanks for changing it to American "neighbor", as the project description contains the American one.